src/compiler/data.ts

```diff
@@ -52,10 +51,16 @@ export function compileData(model: Model): VgData[] {
     def.push(stack.def(model, stackDef));
   }
 
+  // Time domain tables
+
+  dates.defs(model).forEach(dateDef => {
```

Comment:
For multiple line function, should we still use classic function style for consistency? 

(I found this shorter format super useful to do one-liner lambda like in python though)